### PR TITLE
Download script

### PR DESCRIPTION
**Pull request by Peter9192**
This PR moves the download script from `src/wam2layers/utils/get_era5` to `/scripts/download_era5_ml.py` and `/scripts/download_era5_pl.py`. Thus, it splits the script into two separate scripts for model- and pressure levels, and it moves the code *out of the core package* and into a folder called *scripts*. This signals that this is not part of the model, but just a useful thingy that others may benefit from, without creating the expectation that we'll be endlessly maintaining it.

I ran into problems with downloading the ML data, which I think is related to https://confluence.ecmwf.int/display/CUSF/CDS+Service+disruption+expected+starting+8+September+2022+for+5-6+weeks

### CODE REVIEW

**Comment by Peter9192** [comment]
By the way, this creates a directory structure like so:

```
└── 2020
    └── 1
        ├── ERA5_2020-01-01_d2m.nc
        ├── ERA5_2020-01-01_e.nc
        ├── ERA5_2020-01-01_q_pl.nc
        ├── ERA5_2020-01-01_sp.nc
        ├── ERA5_2020-01-01_tcw.nc
        ├── ERA5_2020-01-01_tp.nc
        ├── ERA5_2020-01-01_u10.nc
        ├── ERA5_2020-01-01_u_pl.nc
        ├── ERA5_2020-01-01_v10.nc
        ├── ERA5_2020-01-02_e.nc
        ├── ERA5_2020-01-02_sp.nc
        ├── ERA5_2020-01-02_tcw.nc
        ├── ERA5_2020-01-02_tp.nc
        ├── ERA5_2020-01-03_e.nc
        ├── ERA5_2020-01-03_sp.nc
        ├── ERA5_2020-01-03_tcw.nc
        └── ERA5_2020-01-03_tp.nc
```

Are we sure we don't want to create a separate folder for each day? Otherwise we will end up with >180 files per folder

**Review by chrisweij — APPROVED** [review]
Looks fine for me, I have included a few small comments. On your remark on the file structure, daily directories are fine for me as well, I do not have a real preference. But maybe @Imme1992 has?

File: scripts/download_era5_ml.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    '17:00',
+    '18:00',
+    '19:00',
+    '20:00',
+    '21:00',
+    '22:00',
+    '23:00',
+]
+
+levels = list(range(137))
```
[the remark this answers is not in the document]
Comment (Peter9192) [reply scripts/download_era5_ml.py]: Good idea, done!

File: scripts/download_era5_pl.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    '17:00',
+    '18:00',
+    '19:00',
+    '20:00',
+    '21:00',
+    '22:00',
+    '23:00',
+]
+
+levels = [
```
[the remark this answers is not in the document]
Comment (Peter9192) [reply scripts/download_era5_pl.py:54]: Done

**Pull request merged**